sdk/ml/azure-ai-ml/tests/pipeline_job/unittests/test_pipeline_job_validate.py

```diff
@@ -123,6 +123,20 @@ def test_pipeline_job_type_sensitive_error_message(self):
         del job_dict["jobs"]["hello_world_component_inline_with_schema"]["component"]["environment"]
         errors = pipeline_job._schema_for_validation.validate(job_dict)
         type_sensitive_union_field = pipeline_job._schema_for_validation.dump_fields["jobs"].value_field
+        print(errors)
```

Comment:
why do we need to print here?

Reply:
oh, I forget to remove this
btw, this test will be expected to re-enable in #26538